src/mixins/itext_behavior.mixin.ts

```diff
@@ -531,67 +532,69 @@ export abstract class ITextBehaviorMixin<
   /**
    * Override to customize the drag image
    * https://developer.mozilla.org/en-US/docs/Web/API/DataTransfer/setDragImage
-   * @param {DragEvent} e
-   * @param {object} data
-   * @param {number} data.selectionStart
-   * @param {number} data.selectionEnd
-   * @param {string} data.text
-   * @param {string} data.value selected text
```

Comment:
was this wrong? which is the correct documentation?

Reply:
it is wrong, only selectionStart/End remains

Reply:
scratch that.
those are the only used by the method.